content/blog/2024/01/2024-01-30-2024-roadmap-and-fundraiser.md

```diff
@@ -0,0 +1,78 @@
++++
+date = "2024-01-30T17:00:00Z"
+title = "A roadmap and appeal for help from The Matrix.org Foundation"
+
+[taxonomies]
+author = ["Josh Simmons"]
+category = ["Foundation"]
+
++++
+
+## A new fundraising drive
+
+Today we launch a new fundraising drive, talk about the scope of the Foundation's work, and begin to unpack our emerging roadmap for the future. **There is a lot going on and we need your help to keep it going!**
+
+At the end of 2022 Matthew and Amandine [sounded the alarm](https://matrix.org/blog/2022/12/25/the-matrix-holiday-update-2022/): the Foundation needed more support. To deliver that, they [launched the Foundation's membership program](https://matrix.org/blog/2022/12/01/funding-matrix-via-the-matrix-org-foundation/#introducing-foundation-memberships-and-the-governing-board). They also introduced open governance, and committed to hiring a Managing Director to act as a robust, neutral steward.
```

Comment:
I am not native in English but I thought after a colon it is treated like a new sentence so I expected "the Foundation" to be "The Foundation". If that's however my german grammar leaking feel free to ignore me 🙈

Reply:
I think you're correct about that! This is just a quirk of my style 😂